### PR TITLE
prd: bump node-exporter to 1.3.0

### LINKED ISSUE #1225: Bump node-exporter from v1.1.2 to v1.2.2 or v1.3.0
subtask of #1171
ref #1222

修正対象
https://github.com/cloudnativedaysjp/dreamkast-infra/blob/main/manifests/infra/prometheus/base/node-exporter.yaml
https://github.com/cloudnativedaysjp/dreamkast-infra/blob/main/manifests/infra/prometheus/base/monitor.yaml#L10

マニフェストを自前で持たず、このあたりをそのまま使うこともできそうです。
https://github.com/prometheus-operator/kube-prometheus/tree/v0.9.0/manifests
node-exporter-*.yaml

### PR DESCRIPTION
**Pull request by mayocream**
Fix #1225

### CODE REVIEW

**Comment by inductor** [comment]
@mayocream Please apply changes from https://github.com/cloudnativedaysjp/dreamkast-infra/pull/1236

File: manifests/infra/prometheus/base/monitor.yaml [hunk 1]
```diff
@@ -1,13 +1,13 @@
...
+# https://raw.githubusercontent.com/prometheus-operator/kube-prometheus/v0.9.0/manifests/node-exporter-serviceMonitor.yaml
 apiVersion: monitoring.coreos.com/v1
 kind: ServiceMonitor
 metadata:
   labels:
     app.kubernetes.io/component: exporter
     app.kubernetes.io/name: node-exporter
     app.kubernetes.io/part-of: kube-prometheus
-    app.kubernetes.io/version: 1.1.2
+    app.kubernetes.io/version: 1.2.2
```
Comment (oke-py) [inline manifests/infra/prometheus/base/monitor.yaml:10]:
```suggestion
    app.kubernetes.io/version: 1.3.0
```

**Comment by mayocream** [comment]
@oke-py Thanks!

**Review by oke-py — APPROVED** [review]
Thanks!

**Comment by oke-py** [comment]
@inductor Have you reviewed this PR? Could you check it out and merge it?

**Review by inductor — APPROVED** [review]

**Pull request merged**